### PR TITLE
FEM: source and comment typos

### PR DESCRIPTION
**Pull request by luzpaz**
a8e2ebcfde1e459a1b1645211ecac24a2b8e71b4 contains source changes. Please review carefully. Thanks!

### CODE REVIEW

File: src/Mod/Fem/feminout/importZ88Mesh.py [hunk 1]
```diff
@@ -454,6 +454,6 @@ def get_z88_element_type(femmesh, femelement_table=None):
         print('Edge femmesh will be exported as 3D truss element nr 4')
         return 4
     else:
-        print('Neither, edge, face or solid femmesh')
+        print('Neither edge, face, or solid femmesh')
```
Comment (luzpaz) [inline src/Mod/Fem/feminout/importZ88Mesh.py:457]: What I'm interpreting being conveyed here is 
"Neither edge (nor) face (nor) solid femmesh"
So I don't think the comma should be after `Neither`   
Correct me if I'm wrong.

Comment (berndhahnebach) [reply src/Mod/Fem/feminout/importZ88Mesh.py:457]: you are right! should we gone write the nor to be more clear?

Comment (luzpaz) [reply src/Mod/Fem/feminout/importZ88Mesh.py:457]: 2 ways to write this:
`Neither edge, face, or solid femmesh`  
or  
`Neither edge nor face nor solid femmesh`

**Comment by berndhahnebach** [comment]
merged

**Comment by luzpaz** [comment]
@berndhahnebach not all fixes were in the https://github.com/FreeCAD/FreeCAD/pull/1187 that was done purposefully, right? (just checking)

**Comment by berndhahnebach** [comment]
upps, did I forget some commits? Would you mind making another PR?

**Comment by luzpaz** [comment]
@berndhahnebach sure, can you please rebase femdev ?

**Comment by berndhahnebach** [comment]
done

**Pull request closed without merging**